app/models/job.rb

```diff
@@ -83,6 +83,18 @@ def send_to_lims
     LimsClient.post(lims_url, lims_data)
   end
 
+  def claim_materials!
```

Comment:
Could `claim_materials!` and `release_materials!` be merged into a single method `material_available` that takes true or false as a parameter? Would reduce the repeated code